ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly.yaml

```diff
@@ -690,6 +690,15 @@ tests:
     test:
     - chain: openshift-e2e-test-qe-destructive
     workflow: cucushift-installer-rehearse-aws-usgov-ipi-private-workers-marketplace
+- as: aws-ipi-sno-etcd-encryption-f14
```

Comment:
we want to treat this job as a normal ci job, right? If you just want to cover sno + etcd-encryption case, we already have `aws-ipi-disc-priv-sno` job in multi-nightly.yaml. You can append `etcd-encryption` step to create a new install workflow `aws-ipi-disc-priv-sno-etcd-encryption`, or even, you can directly replace `aws-ipi-disc-priv-sno` with `aws-ipi-sno-etcd-encryption`

Reply:
The workflow ` workflow: cucushift-installer-rehearse-aws-ipi-sno-etcd-encryption` is used for another recert job https://github.com/openshift/release/pull/49183. To add these sno + etcd-encryption test here, I can consider merge them into multi-nightly.yaml in another job.

Reply:
Raised another pr https://github.com/openshift/release/pull/49845 for your suggestion.